client/benchmark/src/main/java/org/elasticsearch/client/benchmark/rest/RestClientBenchmark.java

```diff
@@ -47,10 +47,7 @@ public static void main(String[] args) throws Exception {
 
     @Override
     protected RestClient client(String benchmarkTargetHost) {
-        CloseableHttpClient httpClient = HttpClients
-            .custom()
-            .setRetryHandler(new StandardHttpRequestRetryHandler(3, true))
-            .build();
+        CloseableHttpClient httpClient = HttpClients.createDefault();
```

Comment:
I meant to not provide the http client, it is not a mandatory argument. you can remove this line too.